pkg/storage/secret/metadata/keeper_store.go

```diff
@@ -459,20 +467,20 @@ func (s *keeperMetadataStorage) validateSecureValueReferences(ctx context.Contex
 	return nil
 }
 
-func (s *keeperMetadataStorage) GetKeeperConfig(ctx context.Context, namespace string, name string) (contracts.KeeperType, secretv0alpha1.KeeperConfig, error) {
-	// Check if keeper is default sql.
-	if name == contracts.DefaultSQLKeeper {
-		return contracts.SQLKeeperType, nil, nil
+func (s *keeperMetadataStorage) GetKeeperConfig(ctx context.Context, namespace string, name *string) (secretv0alpha1.KeeperConfig, error) {
```

Comment:
Fetch SecureValue, pass the `Keeper` as-is, it might be `nil` now, we want that to just early return a `nil` config, since we use the "system keeper"